### PR TITLE
Add an option to ban the \n character in chat mode

### PR DESCRIPTION
**Pull request by oobabooga**
This is a guaranteed way to get long replies in chat mode. 

The caveat is that the reply never ends, so it must be stopped early using some criterion.

### CODE REVIEW

**Comment by FartyPants** [comment]
This is a fantastic example of using logits and scores

**Comment by oobabooga** [comment]
Thanks @FartyPants. Getting models to talk more has always been an issue -- that's why the "chat generation attempts" parameter exists, and it never worked very well. This should solve the problem for good.

**Comment by BadisG** [comment]
Doesn't seem to be working for me
WuiTtuX

**Comment by oobabooga** [comment]
Make sure to use a _HF loader. Also try the latest commit https://github.com/oobabooga/text-generation-webui/commit/6c521ce96787552a9604c344b9949945ef359a59

**Comment by BadisG** [comment]
It's... interesting to say the least 😂

https://github.com/oobabooga/text-generation-webui/assets/110173477/1d89509b-bca6-490d-b1da-99e54f7df503

**Comment by oobabooga** [comment]
It can make the character go pretty crazy indeed. I usually leave mine at 80-120 tokens just to avoid 1 phrase replies.

**Comment by guispfilho** [comment]
Hi, is it possible to use it with model loader llama.cpp?
